invoices/invoiceregistry.go

```diff
@@ -486,13 +632,38 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 		}, nil
 
 	case channeldb.HtlcStateSettled:
+		// Also settle any previously accepted htlcs.
```

Comment:
s/accepted/settled ?

Reply:
No, it is correct. An an htlc is now marked as settled, it may be the case that it was previously accepted.

Reply:
Then why skip the HTLC if `htlc.State != channeldb.HtlcStateSettled `? 🤔

Reply:
The invoice state is leading. If an htlc is marked as settled, we should follow now and settle the htlc with our peer. But only if it is indeed marked as settled. Comment added